### DISCUSSION TITLE
Is this mean i can use Coplilot for free?

### DISCUSSION THREAD

**Discussion by bestony** [community/community#19137] [Copilot Conversations] [answered]
<img width="790" alt="image" src="https://user-images.githubusercontent.com/13283837/174958875-26f1e128-6198-4b81-a7b0-1ec1f8476dd0.png">

I don't find any where to verify i can use it free or just a free trial.

**Comment by vedantmgoyal9** [comment, marked as answer]
You can use it for free. It's not a trial.

**Reply by bestony** [reply]
do you a github staff? or just user?

**Reply by vedantmgoyal9** [reply]
I'm a user, but since I'm also being shown the same thing, I've verified that we can use it for free as long as I've GitHub Student Pack.

**Reply by bestony** [reply]
thanks!

**Reply by davecheney** [reply]
If you have a 2020, 2021 or 2022 student coupon you may use copilot for free.

**Comment by fallstarlee-svg** [comment]
Free robucx
